demo/ibis-demo.es.qmd

```diff
@@ -0,0 +1,225 @@
+---
+title: Demo de Ibis
+---
+
+Primero, importar Ibis:
+
+```{python}
+import ibis
+from ibis import *
+```
+
+## Conectar a DuckDB
+
+```{python}
+con = ibis.duckdb.connect()
+```
+
+## Cargar unos datos adentro de las tablas
+
+```{python}
+ratings = con.read_parquet("data/imdb_title_ratings.parquet", table_name="ratings")
+basics = con.read_parquet("data/imdb_title_basics.parquet", table_name="basics")
+```
+
+Podemos presentar los nombres de las tablas
+
+```{python}
+con.list_tables()
+```
+
+Y les renderizamos:
+
+```{python}
+ratings
+```
+
+```{python}
+basics
+```
+
+Ahora, ejecutamos con un limite
+
+```{python}
+ratings.execute(limit=10)
+```
+
+Ibis funciona con muchas formatos en memoría, se incluye Pandas y PyArrow
+
+```{python}
+basics.to_pandas(limit=10)
+```
+
+```{python}
+ratings.to_pyarrow(limit=10)
+```
+
+Y Polars, si usted lo tiene installado.
+
+```{python}
+#to show this we will need to install polars
+#basics.to_polars()
+```
+
+## Properly named columns and interactive mode
+
+A veces, ustedes tienen columnas que tienen nombres sucios, y así que ustedes les necesitan limpiar.
```

Comment:
```suggestion
A veces, ustedes tienen columnas que tienen nombres sucios, y así que necesitan limpiarlos.
```
I'm not sure if the word "sucios" is a good one here, but I haven't come up with another one.